corehq/apps/users/management/commands/set_locations_properly.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.users.models import CommCareUser
+from corehq.apps.es import UserES
+from corehq.apps.locations.models import Location
+from dimagi.utils.couch.database import iter_docs
+
+
+def get_affected_ids():
+    users = (UserES()
+             .mobile_users()
+             .non_null("location_id")
```

Comment:
could there potentially be users with null location ids because they were unassigned from a location but whose user data was never updated?